lib/rubocop/cop/rspec/empty_example_group.rb

```diff
@@ -35,6 +35,30 @@ module RSpec
       #     pending 'will add tests later'
       #   end
       #
+      # @example configuration
+      # # .rubocop.yml
+      # RSpec:
+      #   Language:
+      #     Includes:
+      #       Examples:
+      #         - include_tests
+      
+      # # spec_helper.rb
+      # RSpec.configure do |config|
+      #   config.alias_it_behaves_like_to(:include_tests)
+      # end
+      
+      # # bacon_spec.rb
+      # describe Bacon do
+      #   let(:bacon)      { Bacon.new(chunkiness) }
+      #   let(:chunkiness) { false                 }
+      
+      #   context 'extra chunky' do   # not flagged by rubocop
+      #     let(:chunkiness) { true }
+      
+      #     include_tests 'shared tests'
+      #   end
+      # end
```

Comment:
I think the following fix will address the RuboCop failure in CI.

```suggestion
      # @example configuration
      # # .rubocop.yml
      # RSpec:
      #   Language:
      #     Includes:
      #       Examples:
      #         - include_tests
      #
      # # spec_helper.rb
      # RSpec.configure do |config|
      #   config.alias_it_behaves_like_to(:include_tests)
      # end
      #
      # # bacon_spec.rb
      # describe Bacon do
      #   let(:bacon)      { Bacon.new(chunkiness) }
      #   let(:chunkiness) { false                 }
      #
      #   context 'extra chunky' do   # not flagged by rubocop
      #     let(:chunkiness) { true }
      #
      #     include_tests 'shared tests'
      #   end
      # end
```